tendrl/commons/objects/definition/master.yaml

```diff
@@ -71,6 +71,24 @@ namespace.tendrl:
       type: Create
       uuid: 2f94a48a-05d7-408c-b400-e27827f4edef
       version: 1
+    AddOsds:
```

Comment:
This flow belongs to integrations.ceph.flows in the node agent (https://github.com/Tendrl/node-agent/tree/develop/tendrl/integrations/ceph)

Anything under the ceph_help files are actually intended to be integration specific flows under NS "integrations.$sds_name"  in node-agent